system/core/pages.php

```diff
@@ -195,6 +195,15 @@ function loadPage(RequestContext $O_O) {
 					exit;
 				}
 				break;
+        // hier invoegen
+        case HyphaRequest::HYPHA_SYSTEM_PAGE_SEARCH: //bz search
```

Comment:
This comment (and on the next line) can also be dropped?

Reply:
done